build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
 }
 
 group 'org.labkey.build'
-project.version = "1.31.0-SNAPSHOT"
+project.version = "1.30.1-SNAPSHOT"
```

Comment:
any reason why the minor version number went down here?

Reply:
The current release is `1.30.0`, `1.31.0-SNAPSHOT` is just a placeholder for the next release.

Reply:
got it. that makes sense